### ISSUE TITLE
Wizard for nextjs is using deprecated dependencies

### ISSUE THREAD

**Issue by vgrozdanic** [getsentry/sentry-wizard#658]
If you run a wizard for nextjs, there are 2 deprecation warnings:

![Image](https://github.com/user-attachments/assets/20f2db4f-783c-445f-a4e6-48eff3ba94e8)

**Comment by Lms24** [comment]
Both of the `glob` and the `inflight` warnings would be resolved if we update `glob`. However, `glob@9` requires Node 16 or higher which means we'd need to drop Node 14 support.

We already talked about dropping Node 14 support, ideally directly bumping to Node 18, to break away from EOL versions. I'm going to assign the "Breaking" Label here and put it to the new milestone issue. Cutting a major for the wizard is relatively cheap so we could do it. Just wanna make sure we're aligned within the team that we actually want to do this right now.

**Comment by NickBrimmer** [comment]
Hi @Lms24 any updates on this? I'm seeing the same error when pairing the Wizard with @sentry/node installation with v20.18.0. Just wanting to check :)

**Comment by Lms24** [comment]
Hey @NickBrimmer, no update at the moment. This didn't turn out to be of high priority for us. Is this impacting you negatively?

**Comment by Lms24** [comment]
So turns out we are in fact cutting a wizard major but I might have incorrectly stated that bumping `glob` requires node version support changes. Looks like `glob@9.3.5` requires [Node 14.17.0 or higher](https://github.com/isaacs/node-glob/blob/760ad8bacb9ba3dc66ede885b416968fb41c8685/package.json#L95) and our wizard requires [Node 14.18.0](https://github.com/getsentry/sentry-wizard/blob/6e442046bfdd411724f2f40cb82d609073e63445/package.json#L124)

**Comment by NickBrimmer** [comment]
@Lms24 yes! Our team is implementing Sentry in for a tool that is being used for a few hundred clients. I'm glad to see this moved forward and was resolved! Thank you.
